util/random.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2017 PingCAP, Inc.
```

Comment:
It's just a networking level stub. We can only add test cases after we implement them in **business logic level**, just like we do in `server/{sever.go,conn.go}`.